lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +22,96 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  color: $nx-text-color-light;
+  display: grid;
+  grid-template-columns: 85% auto;
```

Comment:
How'd you arrive at 85%? I would have thought something like `1fr auto` would be better so that the right side of the grid takes no space when it's empty

Reply:
> How'd you arrive at 85%? I would have thought something like `1fr auto` would be better so that the right side of the grid takes no space when it's empty

I had tried `fr` but wasn't getting the results I wanted but I think that's because i wasn't using it with `auto`, you suggestion works very well: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/c5aa6499a051116f2101b2d7be86452d4062f373